OpenEdXMobile/build.gradle

```diff
@@ -259,6 +260,13 @@ def firebaseEnabled = firebaseConfig?.get('ENABLED')
 if (firebaseEnabled ?: false) {
     apply plugin: 'com.google.gms.google-services'
     // Apply the Performance Monitoring plugin to enable instrumentation
+    // Getting some libraries conflict after adding the microsoft client library for microsoft SSO
+    // with message "In project '*' a resolved Google Play services library dependency depends on
+    // another at an exact version (e.g. [1.3.1 ,2.3])." so ad below statement to resolve build issue.
+    // Ref: https://stackoverflow.com/a/57479692
+    googleServices {
+        disableVersionCheck = true
+    }
     apply plugin: 'com.google.firebase.firebase-perf'
```

Comment:
This comment should be above this line:
`// Apply the Performance Monitoring plugin to enable instrumentation`

Reply:
Done.